<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>gtFileDialog</class>
 <widget class="QDialog" name="gtFileDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>410</width>
    <height>320</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Open</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <property name="leftMargin">
    <number>9</number>
   </property>
   <property name="topMargin">
    <number>9</number>
   </property>
   <property name="rightMargin">
    <number>9</number>
   </property>
   <property name="bottomMargin">
    <number>9</number>
   </property>
   <item row="0" column="0" colspan="3">
    <widget class="ScFileWidget" name="fileWidget" native="true"/>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="label">
     <property name="text">
      <string>&amp;Importer:</string>
     </property>
     <property name="buddy">
      <cstring>importerCombo</cstring>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QComboBox" name="importerCombo"/>
   </item>
   <item row="1" column="2">
    <widget class="QCheckBox" name="textOnlyCheckBox">
     <property name="text">
      <string>Import &amp;Text Only</string>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string>&amp;Encoding:</string>
     </property>
     <property name="buddy">
      <cstring>encodingCombo</cstring>
     </property>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="QComboBox" name="encodingCombo"/>
   </item>
   <item row="2" column="2">
    <widget class="QCheckBox" name="prefixStylesCheckBox">
     <property name="text">
      <string>Prefix Styles with Item Name</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="3" column="0" colspan="3">
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>ScFileWidget</class>
   <extends>QWidget</extends>
   <header>ui/scfilewidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
